api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXRetQueries.java

```diff
@@ -1076,6 +1076,505 @@
           + "                           9)\n"
           + "  AND e.location_id=:location";
 
+  private static String artTargetInitiationXGenderAtXToYAge =
```

Comment:
All static and hard corded values for the parameters need to be substituted with parameters that can be passed at run time through the configurations of the global properties. I understand the aim was to include it the same way as it was originally, but for our code quality, we need those substituted

Reply:
actually the reason for not replacing these ATM is that this report may change as TX_TB did and we would end up not using these queries which would mean cleaning all such un-used gps etc